change/@microsoft-teams-js-81b620f2-8af1-4450-89b0-20600bac843e.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Unified app id validation, removed incomplete HTML element transformation, and refreshed how script tags are identified in strings",
```

Comment:
I go back and forth on patch notes like this, because for most external devs they will read a description like this and have no idea what it means for them (which is, I assume, almost nothing).

Is there a punchy sentence that describes which parts of this change an external dev would care about? Maybe something like "Updated internal app id validation"

Reply:
I'm not actually sure any external dev will care about this change at all since it's purely internal infrastructural updates. They shouldn't observe any API or behavior difference at all. I guess the library will be a little smaller now (less duplicated code) and it will be harder to hack, theoretically. Not sure if that makes for good patch notes...

Reply:
I can, of course, change it to "updated internal app id validation" but I'm not sure an external dev would care much about that either. :)